common/persistence/sql/sql_execution_store_test.go

```diff
@@ -2911,7 +2911,7 @@ func TestGetHistoryTasks_SQL(t *testing.T) {
 				},
 			},
 			expectedNextPageToken: func() []byte {
-				ti := &timerTaskPageToken{TaskID: 101, Timestamp: time.Unix(1, 1).UTC()}
+				ti := &timerTaskPageToken{TaskID: 101, Timestamp: time.Unix(1, 1)}
```

Comment:
hopefully completely unnecessary, but to sanity check: did this fail?

if it did, then it might imply some underlying issues, as we definitely should not be server-timezone-sensitive.

Reply:
Yeap, that failed just because of mismatched timezones. I've observed this behaviour earlier in previous projects we you have integration tests, so the test must explicitly specify UTC in provided and expected structures, or not specify at all and rely on local timezone.

Reply:
(to follow up from later reading/chat)
this was due to these page-tokens using _default JSON serialization_, which is... extremely not-good.

_but_ it does mean that it correctly round trips regardless of the timezone, so in practice this is fine.